src/Notifier.ts

```diff
@@ -37,6 +37,7 @@ import RoomViewStore from "./stores/RoomViewStore";
 import UserActivity from "./UserActivity";
```

Comment:
> Would it be possible to rephrase the PR title to have it look like a bug fix in the changelog please?

I'm actually not sure if this is a bug fix, since notifications never had support for virtual -> native user mapping in the first place.

And it is user visible... so I suppose this would be an enhancement, with a changelog entry of "Show details of the native room in notifications when new events occur in virtual rooms."

My only worry is that this will make no sense to 99% of users :yum: